packages/medusa/src/services/return.js

```diff
@@ -374,6 +374,12 @@ class ReturnService extends BaseService {
         refund_amount: Math.floor(toRefund),
       }
 
+
+      const returnReasons = await this.returnReasonService_.listReasonsFromIds(returnLines.map(rl => rl.reason_id))
```

Comment:
See above change regarding `list({ id: idArray })`. 

Also this line is too wide.